pkg/setup.cfg

```diff
@@ -71,7 +71,7 @@ requires = python >= 2.6
            numpy >= 1.7
```

Comment:
this version should match the one in the requirements, which i think can be `>=1.14` for the features actually in use